build/Build.Pack.cs

```diff
@@ -477,12 +477,13 @@ void PackTarballs(string runtimeId)
             $"tentacle-{FullSemVer}-{NetCore}-{runtimeId}.tar.gz");
     }
 
-    void BuildAndPushOrLoadKubernetesTentacleContainerImage(bool push, bool load)
+    void BuildAndPushOrLoadKubernetesTentacleContainerImage(bool push, bool load, string? host = null)
     {
+        host = host is not null ? $"{host}/" : null;
```

Comment:
I would declare a new variable called "hostPrefix" and set it to an empty string instead of `null`

Reply:
Like `var hostPrefix = host is not null ? $"{host}/" : string.Empty;`

Reply:
Done 👍